packages/frontend/src/features/sqlRunner/components/Header/HeaderVirtualView.tsx

```diff
@@ -274,10 +278,13 @@ export const HeaderVirtualView: FC<{
             <Group spacing="xs">
                 <Group spacing="xs">
                     <MantineIcon icon={IconTableAlias} />
-                    <Text fz="sm" fw={500}>
-                        {/* TODO: Allow editing name */}
-                        Editing {virtualViewState.name}
-                    </Text>
+                    Editing
+                    <TextInput
```

Comment:
Should we have a `pencil` icon to toggle this TextInput ?

Reply:
🤷 I think what you have is great already 👍